tests/wpt/web-platform-tests/html/webappapis/scripting/events/iframe-onload.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html>
+<meta charset="utf-8">
+<title></title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<iframe id="blank_iframe"></iframe>
+
+<script>
+  var iframe_loads = 0;
+  function iframeOnLoad(t) {
+    iframe_loads++;
+
+    setTimeout(function() {
```

Comment:
Unfortunately, I am not entirely sure if there is a better option than doing a `setTimeout`.